packages/textlint-tester/src/textlint-tester.ts

```diff
@@ -40,6 +40,53 @@ function assertHasFixer(ruleCreator: any, ruleName: string): any {
     throw new Error(`Not found \`fixer\` function in the ruleCreator: ${ruleName}`);
 }
 
+function assertTestConfig(testConfig: TestConfig): any {
+    assert.notEqual(testConfig, null, "TestConfig is null");
+    assert.notEqual(
+        Object.keys(testConfig).length === 0 && testConfig.constructor === Object,
+        true,
+        "TestConfig is empty"
+    );
+    assert(Array.isArray(testConfig.rules), "TestConfig.rules should be an array");
+    assert.notEqual(testConfig.rules.length, 0, "TestConfig.rules should have at least one rule");
```

Comment:
Prefer to use [assert.notStrictEqual](https://nodejs.org/api/assert.html#assert_assert_notstrictequal_actual_expected_message "assert.notStrictEqual").
Or jsut use `assert.ok(testConfig.rules.length > 0)`

Reply:
Fixed with `assert.ok(testConfig.rules.length > 0)` for simplicity.